src/store/record.js

```diff
@@ -114,6 +116,8 @@ let recordStore = {
             };
             state.sections.generalInformation.data.metadata.support_links = JSON.parse(JSON.stringify(record.support_links));
             state.sections.generalInformation.initialData.metadata.support_links = JSON.parse(JSON.stringify(record.support_links));
+            state.sections.additionalInformation.data.metadata.support_links = JSON.parse(JSON.stringify(record.support_links));
+            state.sections.additionalInformation.initialData.metadata.support_links = JSON.parse(JSON.stringify(record.support_links));
```

Comment:
Instead, I would have taken the support links from sections.dataAccess.initial_data. Both work though.

Reply:
I'm not quite sure what you mean - would you mind giving an example, please?

Reply:
I would have removed these two line and added one line here https://github.com/FAIRsharing/fairsharing.github.io/blob/b2ad7cf5c86f7eef50bf4a2b639e21a2ea8fe740/src/store/record.js#L325
` newRecord.metadata.supportslink = state.sections.dataAccess.initialData.link_supports;`
Both work though.